.github/workflows/reusable.rebuild.yml

```diff
@@ -252,14 +256,17 @@ jobs:
     - name: Build LLVM
       shell: cmd
       run: |
-        python ./alloy.py -b --version=${{ inputs.version }} --verbose --generator="${{ inputs.vs_generator }}"
-        cd alloy_results* && type alloy_build.log
+        mkdir build-${{ inputs.version }}
+        cd build-${{ inputs.version }}
+        cmake %ISPC_HOME%\superbuild --preset os -G "NMake Makefiles" -DLLVM_VERSION=${{ inputs.version }} -DCMAKE_INSTALL_PREFIX=%LLVM_HOME%\bin-${{ inputs.version }} -DBUILD_STAGE2_TOOLCHAIN_ONLY=ON -DXE_DEPS=OFF -DLLVM_DISABLE_ASSERTIONS=OFF
```

Comment:
Why using nmake?
Also instead of running `nmake` (or whatever else build tool) directly, you can do `cmake --build .`, which is generally preferable for more cross-platform style.

Reply:
No specific reason, I just didn't test that with msbuild. Probably, it is OK to use default it, as well ([here](https://github.com/ispc/ispc/actions/runs/6074148124)). 
I have fixed `cmake --build`.